ote/src/cljs/ote/app/controller/service_search.cljs

```diff
@@ -43,7 +45,7 @@
                        :as filters}]
   (merge
    (when-not (empty? oa)
-     {:operation_area (str/join "," (map :text oa))})
+     {:operation_area  (str/join "," (map :value oa))})
```

Comment:
@mattiuusitalo Your editor seems to use different formatting rules as we have as convention in the project.
E.g. here there should be just one space between `:operation_area  (`
Similar diff can be seen in this file startoing around line 382

Reply:
The other diffs were poorly indented originally. The one highlighted here is definitely poor formatting.